crates/router/src/core/routing/helpers.rs

```diff
@@ -273,10 +284,7 @@ pub struct RoutingAlgorithmHelpers<'h> {
 
 #[derive(Clone, Debug)]
 pub struct ConnectNameAndMCAIdForProfile<'a>(
-    pub  FxHashSet<(
-        &'a String,
-        common_utils::id_type::MerchantConnectorAccountId,
-    )>,
+    pub FxHashSet<(&'a String, id_type::MerchantConnectorAccountId)>,
```

Comment:
Why is this `&String` and not `&str`?

Reply:
Not sure on this as this was there previously I have just refactored the imports.